JayJayson84/VSIX.Umbraco.DictionaryBuilder
# Hotfix/1.3.9

# Description

Updates the permalinks for documentation images and repository paths.

### Type of change

Tick the relevant option.


## Where should the reviewer start?

- Clone the branch.

## How should this be tested?

1. Check the correct images are showing in the ReadMe.md file.
2. Check the repository links route to an up-to-date version.
